File: HomeTherapistAdmin/app/Models/Order.php
<?php

namespace App\Models;

use Backpack\CRUD\app\Models\Traits\CrudTrait;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Database\Eloquent\Relations\HasOne;

class Order extends Model
{
    use CrudTrait;
    use HasFactory;

    protected $fillable = [
        "user_id",
        'is_complete',
        'longitude',
        'latitude',
        'age_group',
        'gender',
        'customer_address',
        'customer_phone',
        'customer_ID',
        'start_dt',
    ];

    public function user(): BelongsTo
    {
        return $this->belongsTo(User::class, 'user_id', 'staff_ID');
    }

    public function feedback(): HasOne
    {
        return $this->hasOne(Feedback::class);
    }
    public function order_detail(): HasOne
    {
        return $this->hasOne(OrderDetail::class);
    }
}

File: HomeTherapistApi/Models/Order.cs
using System;
using System.Collections.Generic;
using NetTopologySuite.Geometries;

namespace HomeTherapistApi.Models;

public partial class Order
{
  public ulong Id { get; set; }

  public string UserId { get; set; } = null!;

  public DateTime? StartDt { get; set; }

  public string CustomerId { get; set; } = null!;

  public string CustomerPhone { get; set; } = null!;

  public string CustomerAddress { get; set; } = null!;

  public decimal Latitude { get; set; }

  public decimal Longitude { get; set; }
  public string? Gender { get; set; }

  public string? AgeGroup { get; set; }

  public bool IsComplete { get; set; }

  public DateTime? CreatedAt { get; set; }

  public DateTime? UpdatedAt { get; set; }
  public virtual User User { get; set; } = null!;
  public virtual Calendar Calendar { get; set; } = null!;
  public virtual ICollection<Feedback> Feedbacks { get; set; } = new List<Feedback>();
  public virtual ICollection<OrderDetail> OrderDetails { get; set; } = new List<OrderDetail>();
}
